src/Workspaces/Remote/Core/Services/AssetStorage.cs

```diff
@@ -18,8 +18,8 @@ internal class AssetStorage
     {
         public static readonly AssetStorage Default = new AssetStorage();
 
-        private const int CleanupInterval = 3; // 3 minutes
-        private const int PurgeAfter = 30; // 30 minutes
+        private const int CleanupInterval = 1; // 1 minutes
+        private const int PurgeAfter = 3; // 3 minutes
```

Comment:
and these are cache if data is expired, it will be re-fetched from VS. and since most, if not all, is in primary workspace, these data will be rarely actually used. such as when solution is first created or temporary solution from things like preview window is needed.